### PR TITLE
DB Class Added

### PR DESCRIPTION
**Pull request by AdarshShrivastava**

### CODE REVIEW

File: GoogleAPiAutomationProject/src/main/java/pojo/DBClass.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package pojo;
+
+public class DBClass {
+
```
Comment (adarsh2nd) [inline GoogleAPiAutomationProject/src/main/java/pojo/DBClass.java:4]: Add code here

**Review by adarsh2nd — COMMENTED** [review]
add code

File: GoogleAPiAutomationProject/src/main/java/pojo/DBClass.java (hunk 1 above)
Comment (AdarshShrivastava) [reply GoogleAPiAutomationProject/src/main/java/pojo/DBClass.java:4]: added

**Review by adarsh2nd — COMMENTED** [review]
ok

**Pull request merged**